bentoml/configuration/default_bentoml.cfg

```diff
@@ -64,6 +64,7 @@ tls_root_ca_cert =
 tls_client_key =
 tls_client_cert =
 access_token =
+access_token_header =
```

Comment:
Use `access_token_header = access_token` here as the default value to make sure we don't introduce a breaking change?

Reply:
Would be better to do it in the codes kwargs if we want to ensure its backwards compatible? As a user's config will only be reset to this files of they run bentonl config reset

Reply:
This is a new config key so it doesn't have that issue.

I think you must have noticed that the current config module in BentoML has a flaw that it prevents us from changing the default config value easily because it copies the entire default config file to the local bentoml home. But we are fixing that issue in our upcoming new config system https://github.com/bentoml/BentoML/issues/1374